test/e2e/adminapi_redeployvm.go

```diff
@@ -45,30 +44,48 @@ var _ = XDescribe("[Admin API] VM redeploy action", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(resp.StatusCode).To(Equal(http.StatusOK))
 
-		By("verifying through Azure activity logs that the redeployment happened")
-		// Can be material delays in shipping activity logs, hence healthy 15 min wait.
-		// https://docs.microsoft.com/en-us/azure/azure-monitor/logs/data-ingestion-time#azure-activity-logs-resource-logs-and-metrics
-		err = wait.PollImmediate(10*time.Second, 20*time.Minute, func() (bool, error) {
-			filter := fmt.Sprintf(
-				"eventTimestamp ge '%s' and resourceId eq '%s'",
-				startTime.Format(time.RFC3339),
-				*vm.ID,
-			)
-
-			activityLogs, err := clients.ActivityLogs.List(ctx, filter, "status,operationName")
```

Comment:
@ellis-johnson good stuff! I think can now remove `ActivityLogs` from here:

https://github.com/Azure/ARO-RP/blob/ac41f365905bb57dda206dae3fb59cc14c98ec58/test/e2e/setup.go#L46


And remove whole package [`github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/insights`](https://github.com/Azure/ARO-RP/tree/ac41f365905bb57dda206dae3fb59cc14c98ec58/pkg/util/azureclient/mgmt/insights).

Which in turns removes a workaround for a Azure go SDK issue:

https://github.com/Azure/ARO-RP/blob/ac41f365905bb57dda206dae3fb59cc14c98ec58/pkg/util/azureclient/mgmt/insights/activitylogs.go#L13-L22

Reply:
Created a PR for this: https://github.com/Azure/ARO-RP/pull/1841